tests/integration/docusaurus/connecting_to_your_data/cloud/spark_s3_yaml_example.py

```diff
@@ -0,0 +1,134 @@
+from ruamel import yaml
+
+import great_expectations as ge
+from great_expectations.core.batch import BatchRequest, RuntimeBatchRequest
+from great_expectations.data_context import BaseDataContext
+from great_expectations.data_context.types.base import (
+    DataContextConfig,
+    InMemoryStoreBackendDefaults,
+)
+
+# NOTE: InMemoryStoreBackendDefaults SHOULD NOT BE USED in normal settings. You
+# may experience data loss as it persists nothing. It is used here for testing.
+# Please refer to docs to learn how to instantiate your DataContext.
```

Comment:
One minor knitpick is that this comment is different than the one in `spark_s3_python_example.py`. Could we add the last statement on `# Please refer to docs to learn how to instantiate your DataContext.` to the other file?

Reply:
GREAT catch. Fixed.